raft/tracker/state.go

```diff
@@ -14,11 +14,22 @@
 
 package tracker
 
+// StateType is the state of a tracked follower.
 type StateType uint64
 
 const (
+	// StateProbe indicates a follower whose last index isn't known. Such a
+	// follower is "probed" (i.e. an append sent periodically) to narrow down
+	// its last index. In the ideal (and common) case, only one round of probing
+	// is necessary as the follower will react with a hint. Followers that are
+	// probed over extended periods of time are often offline.
 	StateProbe StateType = iota
+	// StateReplicate is the state steady in which a follower receives a steady
```

Comment:
s/state steady/steady state/

Reply:
Done (refined the sentence a bit while I was there).